### PR TITLE
Fill out README

### PR DESCRIPTION
**Pull request by mccullochk**
README

Resolves [pe1774](https://app.clubhouse.io/parks/story/1774/improve-docs-based-on-gary-s-feedback)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+that park by navigating to the `Details` page. Then if that park is of interest
+the `Favourites` page will help keep it close at hand. Users can also stay
+up-to-date on the access status of parks (open, close, etc.) and any alert or
+advisories they should be aware of before traveling.
+
+---
+
+### 1. Explore
+
+<div style="display:flex;align-items:center;">
```
Comment (chrisfosterelli) [inline README.md]: **suggestion (non-blocking)**: Strip out the HTML and styling. 

These look really good in your screenshot, but I don't think Github will render these for security reasons. You can see the rendered version of your branch here: https://github.com/bcgov/parks-adventure-mobile/blob/418d277169f276ad311b7a866d8183bc431db445/README.md

File: README.md [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+  <p style="width: 400px;margin-left: 50px;">The <code>Explore</code> page shows
+  users nearby parks which offer <strong>popular activities and facilities</strong>.</p>
+</div>
+
+---
+
+### 2. Favourites
+
+<div style="display:flex;align-items:center;">
+  <kbd><img src="./assets/FavouritesPage.PNG" width="175" /></kbd>
```
Comment (chrisfosterelli) [inline README.md]: **suggestion (non-blocking)**: Make all extensions lowercase for files and `src` attributes.

Some of these extensions are capitalized and some aren't, and the actual files are sometimes capitalized and sometimes not. This works on MacOS as their filesystem is case-insensitive but results in some broken image references on the web.

File: README.md [hunk 3]
```diff
@@ -15,40 +77,65 @@ npm run android
...
 
-# Build a standalone APK for your project, signed and ready for submission to
-# the Google Play Store with release-channel set to "prod". It will be available
-# via your expo dashboard.
-yarn run build-prod-android
+We are using React Native so that we can build this application for both iOS
+and Android with a single data set. To create those bundles we must follow two
+similar but slightly different workflows. For both operating systems we need
+Expo to build us the appropriate bundles, an `.apk` file for Android and a
+`.ipa` file for iOS. The following commands are used for that. I've created 4
```
Comment (chrisfosterelli) [inline README.md]: **nitpick (non-blocking)**: Avoid use of first person in README's

README's typically use the "academic 'we'" or focus on statements (e.g. 'There are 4 different comments')

```suggestion
`.ipa` file for iOS. The following commands are used for that. We've created 4
```

**Review by chrisfosterelli — COMMENTED** [review]
Nice pictures! What did you author that with?

I think this all makes sense to me. Some possible improvements to consider attached.

Gary had mentioned:
* Features — What are some of the  key things
* dev dependencies that aren't already in package.json

I think this covers the features, but are there any dev dependencies to note? I'm thinking things like Android Studio / Xcode / MacOS / Node.js version could be what Gary had in mind.

File: README.md (hunk 2 above)
Comment (mccullochk) [reply README.md]: Oops. I thought I had changed all the file extensions. Thanks for catching this.

File: README.md (hunk 1 above)
Comment (mccullochk) [reply README.md]: Darn ☹️

**Comment by mccullochk** [comment]
@chrisfosterelli, thanks for your review. I've added some more information about tooling. Not sure if this is what you were thinking though. I don't know what you mean about macOS. I don't think that is a requirement, nor do I believe you have to be running a specific version if you are using macOS.

**Comment by chrisfosterelli** [comment]
@mccullochk No problem! I think those tools look good 👍 

> I don't know what you mean about macOS. I don't think that is a requirement

I mean to build native iOS apps Apple requires MacOS, since Xcode is MacOS-only. It seems this applies to React Native: https://github.com/facebook/react-native/issues/6240#issuecomment-191205506

Though there are some notes about Expo working around this.. [I took a look at it](https://docs.expo.io/distribution/building-standalone-apps/#4-wait-for-it-to-finish-building) -- looks like `build:ios` actually pushes the code up to Expo's servers to build in a macOS VM! Wow that's neat. Does that mean xcode is required still?

**Comment by mccullochk** [comment]
From my understanding xcode is only required if you're using Transporter to upload the `.ipa` file to App Store Connect or your can to use an iPhone simulator to test. Maybe I should make that more clear on my comment about Xcode.

**Comment by chrisfosterelli** [comment]
I'm not familiar with the iOS deploy process so I'll defer to you 😄 If a Windows user would be able to build, test, and deploy iOS builds to TestFlight then it doesn't need to go in under dependencies. If that's not possible without transporter / xcode / MacOS then it could be noted. I don't think it matters too much as requiring xcode implies requiring MacOS in either case.

**Comment by mccullochk** [comment]
Okay. Thanks for the feedback @chrisfosterelli. I think I've cleared it up a bit. I'm going to call this good and get Gary's feedback.

**Pull request merged**